### ISSUE TITLE
Android want PHONE permissions

### ISSUE THREAD

**Issue by kleinpetr** [kevincaicedo/imei_plugin#7]
When I call getImei in main, the app show permission popup for PHONE permissions....

**Comment by badrussamsi** [comment]
hi @kleinpetr  i have same error, do you have solution for fix this error?

**Comment by kleinpetr** [comment]
No. Now I am using this plugin phttps://pub.dev/packages/device_id

**Comment by badrussamsi** [comment]
If use this plugin https://pub.dev/packages/device_id 
how about handle your permission?

**Comment by kleinpetr** [comment]
No need it

Klein Petr
Tvorba mobilních a webových aplikací
www.kleinpetr.com
+420 606 954 642
petr@kleinpetr.com
IČO: 03771571

Dne po 30. 12. 2019 8:59 uživatel badrussamsi <notifications@github.com>
napsal:

**Comment by yash365** [comment]
i am also getting the same issue, getting following error : 
java.lang.NullPointerException: Attempt to invoke interface method 'void io.flutter.plugin.common.MethodChannel$Result.success(java.lang.Object)' on a null object reference in flutter
